<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>renamer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_oold">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>old names:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="list_old">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>281</width>
      <height>311</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::MultiSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_new">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>new names:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="list_new">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>281</width>
      <height>311</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="button_insert">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>insert by position</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_save">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>450</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_replace">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>450</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>find and replace</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_exit">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>450</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_undo">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>360</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>undo last</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_strip">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>410</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>strip characters</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_format">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>450</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>format numbers</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_filter">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>filter list</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_move">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>360</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>moving files</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_copy">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>380</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>creating copies</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rename_by">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>380</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>rename by:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>button_filter</tabstop>
  <tabstop>button_undo</tabstop>
  <tabstop>button_insert</tabstop>
  <tabstop>button_strip</tabstop>
  <tabstop>button_replace</tabstop>
  <tabstop>button_format</tabstop>
  <tabstop>radioButton_move</tabstop>
  <tabstop>radioButton_copy</tabstop>
  <tabstop>button_save</tabstop>
  <tabstop>button_exit</tabstop>
  <tabstop>list_old</tabstop>
  <tabstop>list_new</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
